magda-registry-api/src/main/scala/au/csiro/data61/magda/registry/AspectPersistence.scala

```diff
@@ -72,7 +72,7 @@ object AspectPersistence extends Protocols with DiffsonProtocol {
       eventId <- Try {
         if (testAspectPatch.ops.length > 0) {
           val event = PatchAspectDefinitionEvent(id, aspectPatch).toJson.compactPrint
-          sql"insert into Events (eventTypeId, userId, data) values (${PatchAspectDefinitionEvent.Id}, 0, $event::json)".updateAndReturnGeneratedKey().apply()
+          sql"insert into Events (eventTypeId, userId, tenantId, data) values (${PatchAspectDefinitionEvent.Id}, 0, $MAGDA_ADMIN_PORTAL_ID, $event::json)".updateAndReturnGeneratedKey().apply()
```

Comment:
So I think aspect definitions need to be multi-tenant as well, just like Records and Aspects.

Reply:
will fix it later.

Reply:
fixed